### DISCUSSION TITLE
miniPaint marketplace?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#355] [Q&A]
Hi.

I like to edit some things with miniPaint. I wanted to know if there is a site or place where I could develop plugins for miniPaint and earn some financial donation. 

Have you all ever thought about creating a website where developers create plugins for miniPaint and get some money from development?

Today there are things like bugbounty where you get paid for certain features in one or more open or closed tools - that would be an alternative or example to what I ask here.

this is my question and not a feature request.

**Comment by viliusle** [comment]
I do not think that it will happen, this is free, open source project.
